integration/apps/geopmbench.py

```diff
@@ -0,0 +1,76 @@
+import os
+import textwrap
+
+from . import apps
```

Comment:
Why not just ```import apps```?

Reply:
That pattern didn't work unless CWD was in the PYTHONPATH.  We probably need to look through all the files and make sure we import in a consistent way.  There are also some differences between python2 and 3.